server/views/partials/prisonInformationTable.njk

```diff
@@ -95,46 +95,7 @@
     </div>
 
     {% block extraScripts %}
-        <script type="text/javascript" nonce="{{ cspNonce }}">
-            function hidePanels() {
-                for (var i = 0; i < panels.length; i++) {
-                    panels[i].setAttribute('hidden', 'true')
-                }
-            }
-
-            function showPanel(panel) {
-                hidePanels()
-                panel.removeAttribute('hidden')
-            }
-
-            function setCurrent(current) {
-                for (var i = 0; i < panels.length; i++) {
-                    navLinks[i].removeAttribute('aria-selected')
-                }
-
-                current.setAttribute('aria-selected', 'true')
-            }
-
-            var navLinks = document.querySelectorAll('[role="tab"]')
-            var panels = document.querySelectorAll('[role="tabpanel"')
-
-            showPanel(panels[0])
-
-            for (var i = 0; i < navLinks.length; i++) {
-                navLinks[i].addEventListener('click', function (e) {
-                    var link = e.target
-                    var id = link
-                        .attributes['aria-controls']
-                        .value
-                    var panel = document.querySelector('#' + id)
-
-                    showPanel(panel)
-                    setCurrent(link)
-
-                    e.preventDefault()
-                })
-            }
-        </script>
+        <script type="text/javascript" nonce="{{ cspNonce }}" src="../../../../../assets/js/tabPanelTableScript.js"></script>
```

Comment:
I think this should just be `/assets`, as it's in the context of the server paths, not the file paths right? Unless Nunjucks does some magic.